primitives/src/currency.rs

```diff
@@ -127,6 +127,7 @@ create_currency_id! {
 		PLM("Plasm", 18) = 7,
 		PHA("Phala", 18) = 8,
 		HDT("HydraDX", 12) = 9,
+		BCG("Bit.Country", 18) = 10,
```

Comment:
Actually can you make this 11 and reserve 10 for Kilt? Just to keep everything consistent

Reply:
Yes, no problem. I already updated to 11.